### PR TITLE
Add rpc-ceph mimic and bionic gates

### PR DESCRIPTION
**Pull request by mvollman**
Add mimic gates for xenial and bionic

### CODE REVIEW

File: rpc_jobs/rpc_ceph.yml [hunk 1]
```diff
@@ -57,6 +100,7 @@
 
     scenario:
       - "functional"
+      - "functional_mimic"
```
Comment (hughsaunders) [inline rpc_jobs/rpc_ceph.yml]: Any reason bluestore_mimic isn't being added here?

Comment (mvollman) [reply rpc_jobs/rpc_ceph.yml]: Having bluestore_mimic here would provide value.  I left it off to reduce the number of gates we are running to conserve testing resources.  I was worried that we may have too many gates and was being conservative for now.  But I am happy to add it in and we can always come back and modify it later if it becomes a problem.

**Comment by mvollman** [comment]
I can add all the gates to the release stage.  I was being conservative by not adding all the gates at once, but I am happy to add them now and we can pull them back later if there are too many.

**Comment by mvollman** [comment]
recheck_CIT_unit_tests

**Comment by mvollman** [comment]
recheck_unit_tests

**Comment by mvollman** [comment]
recheck_unit_tests

**Pull request merged**